pom.xml

```diff
@@ -100,6 +131,11 @@
                 </configuration>
             </plugin>
         </plugins>
+        <resources>
+            <resource>
+                <directory>src/main/resources</directory>
```

Comment:
Was having a heck of a time getting Intellij to see Messages.class - I will remove